api/src/main/java/jakarta/json/JsonValueImpl.java

```diff
@@ -26,8 +26,16 @@
  */
 final class JsonValueImpl implements JsonValue, Serializable {
 
+    /** for serialization */
+    private static final long serialVersionUID = 83723433120886104L;
+
+    /** Type of this JsonValue. */
     private final ValueType valueType;
 
+    /**
+     * Default constructor.
```

Comment:
I used to think that the default constructor is the one with no arguments...

Reply:
Why exactly the only one available cannot be the default one?

Reply:
Yep, I was wrong: https://docs.oracle.com/javase/specs/jls/se7/html/jls-8.html#jls-8.8.9
So there is interpretation, that if defined - even no args one - it's no longer default (in Java).